plio/urls.py

```diff
@@ -62,7 +62,6 @@
 api_router.register(r"organizations", OrganizationViewSet, basename="organizations")
 api_router.register(r"users", UserViewSet, basename="users")
 api_router.register(r"videos", VideoViewSet, basename="videos")
-# https://stackoverflow.com/questions/48548622/base-name-argument-not-specified-and-could-not-automatically-determine-the-name
```

Comment:
why have you deleted this?

Reply:
I thought that comment was not needed. Correcting it